modules/nf-core/gsea/gsea/main.nf

```diff
@@ -42,12 +43,13 @@ process GSEA_GSEA {
     def VERSION = '4.3.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def chip = chip ? "-chip $chip -collapse true" : ''
```

Comment:
I'd prefer we avoid using chip as both channel and string for the command line.
I'd go for something like:
```suggestion
    def chip_command = chip ? "-chip $chip -collapse true" : ''
```